Formula/xcbeautify.rb

```diff
@@ -0,0 +1,21 @@
+class Xcbeautify < Formula
+  desc "Little beautifier tool for xcodebuild"
+  homepage "https://github.com/thii/xcbeautify"
+  url "https://github.com/thii/xcbeautify.git",
+      :tag      => "0.4.1",
+      :revision => "aa8b8795c9309c36d138bada74a73d1b8b16cda8"
+  head "https://github.com/thii/xcbeautify.git"
+  depends_on :xcode => ["10.2", :build]
+
+  def install
+    if MacOS.full_version < "10.14.4"
+      odie "macOS 10.14.4 or above is require to build xcbeautify."
```

Comment:
odie isn't allowed in formula builds. if it doesn't build on at least the majority of the homebrew supported systems it's not a good fit for homebrew core and should be in a seperate tap.

Reply:
Makes sense. It’s sad that to build a Swift tool that uses Swift 5 (which needs Xcode 10.2) we need to have macOS Mojave.